src/reducers/tickets/badge-print-reducer.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import {
+    REQUEST_BADGE_PRINTS,
+    RECEIVE_BADGE_PRINTS
+} from '../../actions/badge-actions'
+import { LOGOUT_USER } from 'openstack-uicore-foundation/lib/security/actions';
+import {epochToMoment} from "openstack-uicore-foundation/lib/utils/methods";
+
+const DEFAULT_STATE = {
+    badgePrints: [],
+    order: 'id',
+    orderDir: 1,
+    totalBadgePrints: 0,
+    term: '',
+    currentPage: 1,
+    lastPage: 1,
+    perPage: 10,
+};
+
+const badgePrintReducer = (state = DEFAULT_STATE, action) => {
+    const { type, payload } = action
+    switch (type) {
+        case LOGOUT_USER: {
+            // we need this in case the token expired while editing the form
+            if (payload.hasOwnProperty('persistStore')) {
+                return state;
+            } else {
+                return { ...state, entity: { ...DEFAULT_ENTITY } };
```

Comment:
DEFAULT_ENTITY is not defined ?
what is the purpose of catching the LOGUT_USER action /

Reply:
Fixed issue with variable not defied.
LOGOUT_USER catch is to keep consistency with all the reducers at logout